ParabolInc/action
# Filter local

TEST
- turn on the `localFilter` release flag
- sign up with a new user. go through the welcome wizard & hit the team dash
- add "tip" to the agenda items
- add "settings" to the agenda items
- in the filter input, type "ettings$". Notice the cards go away 1 by 1 until only 1 card is left. Notice only agenda item 2 remains
